hrjobcontract/views/modalForm.html

```diff
@@ -560,33 +560,55 @@ <h4 class="modal-title">{{copy.title}}</h4>
                 </div>
             </uib-tab>
             <uib-tab heading="Leave">
+              <div ng-controller="FormLeaveCtrl">
                 <div class="row">
-                    <div class="col-xs-12">
-                        <h4>Leave</h4>
-                        <hr>
-                    </div>
+                  <div class="col-xs-12">
+                      <h4>Leave</h4>
```

Comment:
It seems you are using a mix of 4 and 2 spaces for indentation in this file. Please be consistent with 2